<ui version="4.0" >
 <class>FormCadi</class>
 <widget class="QWidget" name="FormCadi" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>602</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>800</width>
    <height>602</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>CADI</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff>../../../../../usr/share/kademar/utils/cadi/img/cadi.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/cadi.png</iconset>
  </property>
  <widget class="QLabel" name="l_fons" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="pixmap" >
    <pixmap>../../../../../usr/share/kademar/utils/cadi/img/bg/cadi.png</pixmap>
   </property>
   <property name="scaledContents" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStackedWidget" name="pages" >
   <property name="geometry" >
    <rect>
     <x>240</x>
     <y>80</y>
     <width>491</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex" >
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab_principal" >
    <widget class="QFrame" name="frame" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>80</y>
       <width>441</width>
       <height>371</height>
      </rect>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="l_ver" >
      <property name="geometry" >
       <rect>
        <x>150</x>
        <y>73</y>
        <width>271</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text" >
       <string>versio</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_tipu" >
      <property name="geometry" >
       <rect>
        <x>150</x>
        <y>53</y>
        <width>271</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text" >
       <string>tipus</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_kdm" >
      <property name="geometry" >
       <rect>
        <x>150</x>
        <y>33</y>
        <width>271</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text" >
       <string>kdm</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_ram" >
      <property name="geometry" >
       <rect>
        <x>70</x>
        <y>250</y>
        <width>361</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text" >
       <string>RAM</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>35</y>
        <width>109</width>
        <height>85</height>
       </rect>
      </property>
      <property name="text" >
       <string/>
      </property>
      <property name="pixmap" >
       <pixmap>../../../../../usr/share/kademar/utils/cadi/img/pc.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="l_kern" >
      <property name="geometry" >
       <rect>
        <x>150</x>
        <y>92</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text" >
       <string>kernel</string>
      </property>
      <property name="textFormat" >
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_9" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>240</y>
        <width>51</width>
        <height>48</height>
       </rect>
      </property>
      <property name="text" >
       <string/>
      </property>
      <property name="pixmap" >
       <pixmap>../../../../../usr/share/kademar/utils/cadi/img/mem.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="l_cpuinfo" >
      <property name="geometry" >
       <rect>
        <x>70</x>
        <y>184</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text" >
       <string>CPU info</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_cpu" >
      <property name="geometry" >
       <rect>
        <x>70</x>
        <y>164</y>
        <width>361</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text" >
       <string>CPU</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_host" >
      <property name="geometry" >
       <rect>
        <x>150</x>
        <y>110</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text" >
       <string>hostname</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>150</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text" >
       <string/>
      </property>
      <property name="pixmap" >
       <pixmap>../../../../../usr/share/kademar/utils/cadi/img/cpu.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_10" >
      <property name="geometry" >
       <rect>
        <x>130</x>
        <y>13</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font" >
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text" >
       <string>Sistema:</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>20</y>
       <width>441</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <family>DejaVu Sans</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="text" >
      <string>CADI: The Configuration Center</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_software" >
    <widget class="QPushButton" name="b_synaptic" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>70</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text" >
      <string>Synaptic - Install/Remove Programs</string>
     </property>
     <property name="icon" >
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/cadi/img/synaptic.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/synaptic.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="b_preferencies_sistema" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>150</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text" >
      <string>General System Preferences</string>
     </property>
     <property name="icon" >
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/cadi/img/preferences_system_login.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/preferences_system_login.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_4" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>10</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Software</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_hardware" >
    <widget class="QLabel" name="label_5" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>10</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Hardware</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="b_bluetooth" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>150</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text" >
      <string>Bluetooth</string>
     </property>
     <property name="icon" >
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/cadi/img/bluetooth.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/bluetooth.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="b_impressores" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>70</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text" >
      <string>Printers</string>
     </property>
     <property name="icon" >
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/cadi/img/printer1.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/printer1.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="b_kinfocenter" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>310</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text" >
      <string>System Information</string>
     </property>
     <property name="icon" >
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/cadi/img/system-information.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/system-information.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="b_teclats_multimedia" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>230</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text" >
      <string>Multimedia Keyboards</string>
     </property>
     <property name="icon" >
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/cadi/img/keyboard.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/keyboard.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="b_ndiswrapper" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>390</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text" >
      <string>Ndiswrapper - Wifi Not Recognized</string>
     </property>
     <property name="icon" >
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/cadi/img/network-wireless.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/network-wireless.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_xarxa" >
    <widget class="QPushButton" name="b_internet" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>90</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text" >
      <string>Internet and Connectivity</string>
     </property>
     <property name="icon" >
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/cadi/img/network.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/network.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="b_modem" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>170</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text" >
      <string>Modem</string>
     </property>
     <property name="icon" >
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/cadi/img/router.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/router.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_6" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>10</y>
       <width>351</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Network</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="b_pppoe" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>250</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text" >
      <string>ADSL/PPoE</string>
     </property>
     <property name="icon" >
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/cadi/img/pppoe.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/pppoe.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="b_gprs" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>330</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text" >
      <string>GPRS</string>
     </property>
     <property name="icon" >
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/cadi/img/gprs.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/gprs.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_sistema" >
    <widget class="QLabel" name="label_7" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>10</y>
       <width>351</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>System</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="b_grub" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>250</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text" >
      <string>Restauració d'engegada - GRUB</string>
     </property>
     <property name="icon" >
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/cadi/img/hdd.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/hdd.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="b_services" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>170</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text" >
      <string>System Services</string>
     </property>
     <property name="icon" >
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/cadi/img/services.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/services.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="b_users" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>400</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text" >
      <string>Manage Users</string>
     </property>
     <property name="icon" >
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/cadi/img/users.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/users.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="b_language" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>330</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text" >
      <string>System Language Configuration</string>
     </property>
     <property name="icon" >
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/cadi/img/locale.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/locale.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="b_display_configuration" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>90</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text" >
      <string>Display Configuration</string>
     </property>
     <property name="icon" >
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/cadi/img/display.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/display.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="l_kademar" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>10</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="pixmap" >
    <pixmap>../../../../../usr/share/kademar/utils/cadi/img/kademar.png</pixmap>
   </property>
   <property name="scaledContents" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="b_software" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>170</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text" >
    <string>Software</string>
   </property>
   <property name="icon" >
    <iconset>
     <normaloff>../../../../../usr/share/kademar/utils/cadi/img/software.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/software.png</iconset>
   </property>
   <property name="iconSize" >
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="b_hardware" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>230</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text" >
    <string>Hardware</string>
   </property>
   <property name="icon" >
    <iconset>
     <normaloff>../../../../../usr/share/kademar/utils/cadi/img/hardware.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/hardware.png</iconset>
   </property>
   <property name="iconSize" >
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="b_xarxa" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>290</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text" >
    <string>Xarxa</string>
   </property>
   <property name="icon" >
    <iconset>
     <normaloff>../../../../../usr/share/kademar/utils/cadi/img/xarxa.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/xarxa.png</iconset>
   </property>
   <property name="iconSize" >
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="b_sistema" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>350</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text" >
    <string>Sistema</string>
   </property>
   <property name="icon" >
    <iconset>
     <normaloff>../../../../../usr/share/kademar/utils/cadi/img/sistema.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/sistema.png</iconset>
   </property>
   <property name="iconSize" >
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="b_root" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>540</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Administrador</string>
   </property>
   <property name="icon" >
    <iconset>
     <normaloff>../../../../../usr/share/kademar/utils/cadi/img/root.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/root.png</iconset>
   </property>
   <property name="iconSize" >
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>460</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Ejecutando en:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>480</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Modo Usuario</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_sortir" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>540</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Exit</string>
   </property>
   <property name="icon" >
    <iconset>
     <normaloff>../../../../../usr/share/kademar/utils/cadi/img/exit.png</normaloff>../../../../../usr/share/kademar/utils/cadi/img/exit.png</iconset>
   </property>
   <property name="iconSize" >
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
